src/com/encora/javaAssignments/ControlFlowStatements/IfStatement.java

```diff
@@ -0,0 +1,20 @@
+package ControlFlowStatements;
+
+public class IfStatement {
+
+    //Program for If Statement:
+
+    public static void student(int rank) {
+
+        if (rank < 0) {
+            System.out.println("The number is Negative");
+        }
+        System.out.println("The number is Positive");
+    }
```

Comment:
For negative input both the statements will get printed, please change the logic